packages/lib/bookingSuccessRedirect.ts

```diff
@@ -185,7 +194,7 @@ export const useBookingSuccessRedirect = () => {
           ...query,
           ...bookingExtraParams,
         },
-        searchParams: new URLSearchParams(searchParams.toString()),
+        searchParams: filterEmbedParameter(searchParams, isEmbed),
```

Comment:
Instead of making it embed specific we should handle removing the param in `getBookingRedirectExtraParams` and unit test that function as that function has the responidbility of building what params to send